postgres-derive/src/overrides.rs

```diff
@@ -30,6 +31,13 @@ impl Overrides {
                             return Err(Error::new_spanned(&meta.path, "unknown override"));
                         }
 
+                        if overrides.transparent {
+                            return Err(Error::new_spanned(
```

Comment:
Might be best to have some error checking at the end instead of having each branch here check.

Reply:
I moved the check to `fromsql.rs` & `tosql.rs`.